deploy/olm-catalog/service-telemetry-operator/metadata/properties.yaml

```diff
@@ -0,0 +1,4 @@
+    olm.properties: '[{"type": "olm.maxOpenShiftVersion", "value": "4.10"}]'
+properties:
+  - type: olm.maxOpenShiftVersion
+    value: "4.10"
```

Comment:
Looks like this was wrong for a bit. It should be 4.12 in master. Will make a suggested change and commit it.

Reply:
Ummmm weird... it looked like it was wrong wherever I was looking at this before, but that's not true. Fixing my typo. Also that `olm.properties` line was for reference when editing this file, and is invalid.